src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/02-linux-installation.md

```diff
@@ -0,0 +1,263 @@
+
+## Installing Necessary Packages
+
+### Update your system
+
+To start, open up a terminal with the keys `Ctrl + Alt + T`, then type or use `Ctrl + Shift + p` to copy following commands:
+
+```bash
+sudo apt update && sudo apt upgrade -y
+```
+
+![Terminal](/splashkit/linux-installation-fig2.png)
+
+### Install curl and git
+
+Curl and git can be installed with the following command:
+
+```bash
+sudo apt-get install curl git
+```
+
+![Terminal](/splashkit/linux-installation-fig3.png)
+
+## Install SplashKit
+
+### Collect the Installation scripts for SplashKit
+
+Run the following command to download the SplashKit installation scripts:
+
+```bash
+bash <(curl -s https://raw.githubusercontent.com/splashkit/skm/master/install-scripts/skm-install.sh)
+```
+
+![Terminal](/splashkit/linux-installation-fig4.png)
+
+Then you must **RESTART** your terminal.
+
+### skm Command
+
+To install SplashKit, you can use the `skm` command. Run the following command to install SplashKit:
+
+```bash
+skm
+```
+
+If this is successful the output should look like:
+
+```bash
+SplashKit is installed successfully!
+Missing skm command. For help use 'skm help'
+```
+
+![Terminal](/splashkit/linux-installation-fig5.png)
+
+### skm Linux Install
+
+To install SplashKit on Linux, run the following command:
+
+
+```bash
+skm linux install
+```
+
+This may require additional dependencies to be installed, if this is the case you will be prompted to enter your password to install the dependencies.
+
+If this command is successful, the output should look like:
+
+```bash
+Splashkit Installed
+Linux Command run successfully
+```
+
+![Terminal](/splashkit/linux-installation-fig6.png)
+
+
+![Terminal](/splashkit/linux-installation-fig7.png)
+
+### skm Global Install
+
+To install SplashKit globally, run the following command:
+
+```bash
+skm global install
+```
+
+If this command is successful, the output should look like:
+
+```bash
+Splashkit is installed correctly!
+Done
+Global command run successfully
+```
+
+![Terminal](/splashkit/linux-installation-fig8.png)
+
+## Installing Visual Studio Code
+
+Next you will need to install visual studio code, if you have this already installed, then move to installing the necessary extensions for Visual Studio Code.
+
+To install Visual Studio Code, go to the [Visual Studio Code website](https://code.visualstudio.com/) and download the `.deb` file.
+
+![Terminal](/splashkit/linux-installation-fig9.png)
+
+![Terminal](/splashkit/linux-installation-fig10.png)
+
+### Install Visual Studio Code
+
+To install Visual Studio Code, move to the directory where the downloaded `.deb` file is (this is typically your downloads folder), by doing:
+
+```bash
+cd ~/Downloads
+```
+
+Then run the following command to install the `.deb` file:
+
+```bash
+sudo dpkg -i code_1.*.deb
+```
+
+If this command is successfully you will see all processed triggers completed.
```

Comment:
Language. 

_If this command is successful, you will see that all processed triggers have been completed._

Reply:
fixed.